data/transactions/logic/Makefile

```diff
@@ -7,15 +7,13 @@ DOCS :=  ../../../../docs
 
 TEAL_opcodes.md teal.tmLanguage.json:	fields_string.go ../../../cmd/opdoc/opdoc.go ../../../cmd/opdoc/tmLanguage.go eval.go assembler.go doc.go opcodes.go
 	go run ../../../cmd/opdoc/opdoc.go ../../../cmd/opdoc/tmLanguage.go
+	@cp `ls TEAL_opcodes_v??.md | tail -1` TEAL_opcodes.md
 	@if [ -e $(SPECS)/dev/TEAL_opcodes.md ]; then \
 		sed '/^$$/q' $(SPECS)/dev/TEAL_opcodes.md | cat - TEAL_opcodes.md > opcodes.spec; \
 		mv opcodes.spec $(SPECS)/dev/TEAL_opcodes.md; \
 		echo "TEAL_opcodes.md updated in specs repo"; \
 	fi
-	@if [ -e $(DOCS)/docs/reference/teal/opcodes.md ]; then \
-		sed 's/^# /title: /g' TEAL_opcodes.md > $(DOCS)/docs/reference/teal/opcodes.md; \
```

Comment:
we are not copying opcodes.md and specification.md to the docs repo anymorem, why so?

Reply:
The docs people seem to have come up with there own workflow for getting the updates.  That was my personal workflow, where I would have the docs workflow checkout out next to go-algorand, and this copy would help me prepare a PR.